toolkit/scripts/sodiff/sodiff.repo

```diff
@@ -0,0 +1,19 @@
+[mariner-official-base]
```

Comment:
Please use the files from the toolkit or from the mariner-repos folder rather than duplicating it.